runtime/lua/vim/lsp/codelens.lua

```diff
@@ -44,9 +44,10 @@ end
 
 --- Return all lenses for the given buffer
 ---
+---@param bufnr number  Buffer number. 0 can be used for the current buffer.
```

Comment:
nit: the documentation convention is:

    ---@param bufnr number  Buffer number, or 0 for current buffer

Reply:
Sorry, was too quick with the merge.

I can do a follow up to streamline the docs, there are a couple of other places where we use different ways to describe the bufnr parameter.